modules/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -72,28 +72,41 @@ trait SeqexecEngine[F[_]] {
     q:           EventQueue[F],
     id:          Observation.Id,
     user:        UserDetails,
+    observer:    Observer,
```

Comment:
Wouldn't have made sense to include the name used for Observer in `UserDetails` ? Something like `final case class UserDetails(username: String, displayName: String, observerName: String)` And then `observerName` would be the same as `displayName`, until the user writes something different.

Reply:
In hindsight that's probably correct, I'm not sure if there is enough time to fix it